src/main/java/core/basesyntax/service/Writer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface Writer {
+    boolean write(String report, String path);
```

Comment:
https://en.wikipedia.org/wiki/Dependency_inversion_principle
`report` name
abstraction (`Writer`) depends on implementation (`WriteCsvService`). Your implementation told its abstraction that the first parameter would be some `report`. But what if you have some other implementation that would want to write an `essay` for example? So this Writer should be abstracted from its implementation; it shouldn't choose to know what they will want to write to the file. You should choose a more generic name, for example, `content`. It will mean that everyone who wants implement this interface will have to implement `write` method, which writes some generic `content` to some file

Reply:
Please, pay attention to this comment as well